src/spec/intersection_spec.js

```diff
@@ -0,0 +1,18 @@
+import chai from 'chai'
+import intersection from '../lib/intersection.js'
+const expect = chai.expect
+
+describe('Intersection', () => {
+  it('Creates an array of unique values that are included in all given arrays', () => {
+    expect(intersection([4,2,7,5],[4,3,8,6])).to.eql([4])
+  })
+  it('Creates an array of unique values that are included in all given arrays', () => {
+    expect(intersection([6,10,7],[4,3,8,6])).to.eql([6])
+  })
+  it('Creates an array of unique values that are included in all given arrays', () => {
+    expect(intersection([8,10],[4,3,6])).to.eql([])
+  })
+  it('Creates an array of unique values that are included in all given arrays', () => {
```

Comment:
again, this is testing the same case as the first two unit tests. I would love to see tests for what happens if the values in the array are strings, objects, or other arrays. Also worth testing: what if only one array is supplied? What if there is a null value in one or both arrays?